src/sentry/issues/auto_source_code_config/task.py

```diff
@@ -41,7 +41,9 @@ class DeriveCodeMappingsErrorReason(StrEnum):
     EMPTY_TREES = "The trees are empty."
 
 
-def process_event(project_id: int, group_id: int, event_id: str) -> list[CodeMapping]:
+def process_event(
+    project_id: int, group_id: int, event_id: str
+) -> tuple[list[CodeMapping], list[str]]:
```

Comment:
We return the list of in-app stack trace rules created in order to make it easy to test.